android-sdk/src/main/java/webtrekk/android/sdk/util/Util.kt

```diff
@@ -65,4 +66,16 @@ val currentTimeStamp: Long
     inline get() = System.currentTimeMillis()
 
 val currentWebtrekkVersion: String
-    inline get() = BuildConfig.VERSION_NAME
\ No newline at end of file
+    inline get() = BuildConfig.VERSION_NAME
+
+fun jsonToMap(params: String): Map<String, String> {
```

Comment:
It's more idiomatic to make `jsonToMap` as an extension function in `Extension.kt`. This `Util.kt` is usually have properties related to the Android specifically.